api/api.go

```diff
@@ -1,39 +1,155 @@
 package api
 
 import (
-	"github.com/mesg-foundation/core/container"
+	"fmt"
+
 	"github.com/mesg-foundation/core/database"
+	"github.com/mesg-foundation/core/event"
+	"github.com/mesg-foundation/core/execution"
+	"github.com/mesg-foundation/core/pubsub"
+	"github.com/mesg-foundation/core/service"
+	uuid "github.com/satori/go.uuid"
 )
 
 // API exposes all functionalities of MESG core.
 type API struct {
-	db        database.ServiceDB
-	execDB    database.ExecutionDB
-	container container.Container
+	db     database.ServiceDB
+	execDB database.ExecutionDB
+	sm     service.Manager
 }
 
-// Option is a configuration func for MESG.
-type Option func(*API)
-
 // New creates a new API with given options.
-func New(db database.ServiceDB, execDB database.ExecutionDB, options ...Option) (*API, error) {
-	a := &API{db: db, execDB: execDB}
-	for _, option := range options {
-		option(a)
-	}
-	if a.container == nil {
-		var err error
-		a.container, err = container.New()
-		if err != nil {
+func New(db database.ServiceDB, execDB database.ExecutionDB, sm service.Manager) *API {
+	return &API{
+		db:     db,
+		execDB: execDB,
+		sm:     sm,
+	}
+}
+
+// GetService returns service serviceID.
+func (api *API) GetService(service string) (*service.Service, error) {
+	s, err := api.db.Get(service)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := api.sm.Status(s); err != nil {
```

Comment:
Just as a note for future development, the Status should be saved and update directly in the service database.

Reply:
not in the service database, we should have another database/resource that has the records `serviceHash => status`

the service database should contains only the data of the services not their states